technical-docs/designs/repos-merge.md

```diff
@@ -0,0 +1,285 @@
+# Moving SPK to Bedrock Repo: A Phased Approach
+
+## Phase 1: Make Changes to Bedrock and SPK Repos
+
+### Changes to Bedrock Repo
+
+By making following changes in existing Bedrock repo minimizes the number of
+changes after the merge and limits the testing scope.
+
+#### Current Bedrock Repo
+
+```
+├── .github
+├── cluster
+├── docs
+├── gitops
+├── pipelines
+├── test
+├── tools
+├── LICENSE
+├── README.md
+├── .gitignore
+├── azure-pipelines.yml
+```
+
+#### Proposed Changes
+
+- Create `src` folder in `cluster` folder
+- Move all files and folders except `src` from `cluster` to `cluster\src`
+- Move `azure-pipelines.yml` file from the root to `cluster\build\pipelines`
+  directory and make sure all pipelines still works.
+- Move `docs` folder to `cluster` folder and make sure all doc links works.
```

Comment:
@sarath-p last week the spk_docs branch was merged into master branch of Bedrock. Everything in the `docs` folder is `spk` related so we don't need to move `docs` to `cluster`